crates/sui-types/src/error.rs

```diff
@@ -461,10 +461,8 @@ pub enum SuiError {
         authority: AuthorityName,
         reason: String,
     },
-    #[error("Storage error")]
-    StorageError(#[from] TypedStoreError),
-    #[error("Non-RocksDB Storage error: {0}")]
-    GenericStorageError(String),
+    #[error("Storage error: {0}")]
+    StorageError(String),
```

Comment:
aren't these edits going to change the serialization format? this would break grpc, right?

Reply:
I don't think we need any tricky rollouts - just rename deleted fields as deprecated as necessary, and add new fields at the end

Reply:
Ah makes sense ... the bcs deserialization would just ignore the new enum types at the end.

Reply:
Thanks for the discussion, will make this change